### PR TITLE
Ported taar_locale to use clients_daily instead of using longitudinal table

### PR DESCRIPTION
**Pull request by crankycoder**
This migrates the taar_locale job to use the clients_daily table from the deprecated longitudinal table.

### CODE REVIEW

**Comment by crankycoder** [comment]
@mlopatka  r?

**Comment by crankycoder** [comment]
@acmiyaguchi the test failure looks like it's not related to the code in this patch.    It looks like boto is trying to access S3 credentials for some reason.

**Comment by mreid-moz** [comment]
@crankycoder you should be able to rebase on master (after #271) to fix this test failure.

File: mozetl/taar/taar_locale.py [hunk 1]
```diff
@@ -37,26 +36,33 @@ def get_addons(spark):
...
     return spark.sql("""
-        WITH sample AS (
-        SELECT client_id,
-        settings[0].locale AS locality,
-        EXPLODE(active_addons[0])
-        FROM longitudinal
-        WHERE normalized_channel='release'
-          AND build IS NOT NULL
-          AND build[0].application_name='Firefox'
+        WITH sample AS
```
Comment (Dexterp37) [inline mozetl/taar/taar_locale.py]: nit: this should probably be indented with the lower `(`

Comment (crankycoder) [reply mozetl/taar/taar_locale.py]: Doh. I've pushed a commit to clean up that whitespace.

**Comment by mreid-moz** [comment]
Let's hold off merging this until next week due to the allhands code freeze.

**Pull request merged**